go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/stretchr/testify v1.10.0
 	github.com/testcontainers/testcontainers-go v0.35.0
 	github.com/testcontainers/testcontainers-go/modules/postgres v0.35.0
-	github.com/xataio/pg_query_go/v6 v6.0.0-20250124115938-4fa82ad6d036
+	github.com/xataio/pg_query_go/v6 v6.0.0-20250226124908-814e464c6798
```

Comment:
It has to be updated after my PR is merged in pg_query_go: https://github.com/xataio/pg_query_go/pull/9

Reply:
Updated.